### ISSUE TITLE
[Custom fonts module] Integrate with Elementor

### ISSUE THREAD

**Issue by rodica-andronache** [Codeinwp/themeisle-companion#604]
Right now, the custom fonts are integrated just with Neve's typography options from Customizer https://docs.themeisle.com/article/1311-custom-fonts-orbit-fox-module

An idea would be to have the custom fonts accessible from Elementor as well.

Source: Neve Facebook group

**Comment by selul** [comment]
we should make sure that it works with all page builders that we support + otter.

**Comment by cristian-ungureanu** [comment]
@selul I tried to add the custom fonts in Brizy builder but it seems impossible, they have their own mechanism of adding fonts and it's available in their PRO version. They store the fonts in a CPT but they only display the fonts that are uploaded through their interface so inserting a font in that CPT doesn't make the font available in their editor.
Do you want me to have a look at Divi and Thrive too?
I can integrate the custom fonts module with Otter, I'm working at this right now.

**Comment by selul** [comment]
Ok, it's fine, we can skip it. For Divi/Thrive we can add it only is not that complicated.

**Comment by cristian-ungureanu** [comment]
I will close this issue after today's dev meeting. We already have integration with Elementor and Beaver Builder. I tried adding compatibility in Brizy, but I had no success, they have their own mechanism, and adding a font is a feature that Brizy PRO has. I've also tried adding compatibility with Otter. Still, it requires some effort telling Otter's enqueueing font function that the font we're adding it's not Google, so just a filter wouldn't be enough.
